vignettes/teal-data.Rmd

```diff
@@ -30,35 +33,35 @@ my_data <- within(
   }
 )
 
-# get variables from teal_data environment
+# get objects stored in teal_data
 my_data[["data1"]]
 my_data[["data1"]]
 
 # get reproducible code
 get_code(my_data)
 
-# set/get datanames
+# get or set datanames
 datanames(my_data) <- c("data1", "data2")
 datanames(my_data)
 
-# concise print
+# print
 print(my_data)
 ```
 
 
 ## `teal_data` characteristics
 
-`teal_data` class object keeps the following information:
+A `teal_data` object keeps the following information:
 
-- `env` - an environment containing data. 
-- `code` - a string containing code to reproduce `env` (details in [teal_data reproducibility ](teal-data-reproducibility.html)).
-- `datanames` - a character vector with names of datasets.
+- `env` - an environment containing data.
+- `code` - a string containing code to reproduce `env` (details in [reproducibility](teal-data-reproducibility.html)).
+- `datanames` - a character vector listing objects of interest to `teal` modules (details in [this `teal` vignette](https://insightsengineering.github.io/teal/latest-tag/articles/including-data-in-teal-applications.html)).
```

Comment:
Probably the best solution for this "datanames" problem

Reply:
Best I was able to come up with. I am relieved that you approve.